ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/ContractPayloadStore.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.v2
+
+import com.daml.lf.value.Value.{ContractId, VersionedContractInstance}
+
+import scala.concurrent.Future
+
+trait ContractPayloadStore {
+
+  def loadContractPayloads(
```

Comment:
@rgugliel-da wdyt? 
I went for the batched to accommodate implementation for non-canton.
In result there is deserialized value now. We could have here Array[Byte] or proto too, depends on how we want to locate deserialization code/logic/caching etc.